protocols/enocean/README.md

```diff
@@ -0,0 +1,73 @@
+Functional and technical descriptions:
+
+- The OSGi EnOcean Base Driver enables applications running on an OSGi software platform to discover and control EnOcean devices. EnOcean is a radio protocol used in Smart Home and Building markets. OSGi is an execution environment above Java virtual machine to run modular applications.
+
+- The OSGi EnOcean Base Driver implements OSGi standard specification entitled EnOcean Device Service. OSGi EnOcean Device Service specification defines the Java API to discover and control EnOcean devices on the OSGi platform and according to OSGi service design patterns. This API maps the representation model of EnOcean entities defined by EnOcean Equipment Profiles standard into Java classes. OSGi service design patterns are used on the one hand for dynamic discovery, control and eventing of local and networked devices and on the other hand for dynamic network advertising and control of local OSGi services.
+
+- The EnOcean Base Driver is entirely written in Java and uses serial communication drivers usually available on common OS, e.g., Linux. To work properly, the software needs to be installed on an OSGi platform R4 v4.0 (and above) running on a JVM 1.4 (and above) with RXTX Java library. It runs above Linux systems connected to an EnOcean TCM300 USB Gateway delivered as an internal chip or a USB dongle.
+
+
+Tree
+----
+    .
+    - driver-rx-tx                     ; see https://github.com/Orange-OpenSource/Driver-RX-TX
+                                       ; It's a generic interface bundle to the Serial port.
+ 
+    - com.orange.enocean.eeps          ; This bundle is an OSGi EnOcean bundle that contained the
+                                       ; three EEPs (i.e. A5-02-05, D5-00-01, and F6-02-01) that 
+                                       ; the EnOcean Alliance allowed the project to use.
+ 
+    - com.orange.impl.service.enocean  ; This bundle is the EnOcean Base Driver itself. This 
+                                       ; version contains an EnOceanHost Implementation that 
+                                       ; is able to communicate over the serial port.
+                                       ; see https://github.com/osgi/design/tree/master/rfcs/rfc0199
+ 
+    - com.orange.sample.enocean.client ; This is a very simple sample application for using the 
+                                       ; EnOcean base driver. It subscribes to both service and 
+                                       ; event notifications regarding EnOcean events on OSGi.
+ 
+    - cnf                              ; Bnd's repo (contains, among others, the OSGi RFC199 API 
```

Comment:
Hm, this folder is a problem - the contribution should NOT contain any binary jar files, but instead there should be a Maven build that aggregates the required dependencies. We will also need a detailed list of all dependencies as all of them need to be approved by the IP team as well.